app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div id="recipes">
+  <% @recipes.each do |recipe| %>
+    <%= render recipe %>
+  <% end %>
```

Comment:
This works almost perfect! Let's add a lin_to so we can open each recipe in a new page (recipes#show)
Ex:
`<% @recipes.each do |recipe| %>`
`  <%= link_to recipe_path(recipe.id) do %>`
 `    <%= render recipe %>`
`  <% end %>`
`<% end %>`